src/init/isAdminInit.js

```diff
@@ -0,0 +1,16 @@
+export var isAdminInit;
+
+isAdminInit = function(c) {
+    var t = this,
+        appManager = c.appManager,
+        requestManager = c.requestManager,
+        apiPath = appManager.getApiPath();
+
+    return {
+        baseUrl: appManager.getPath() + '/api/me/authorization/ALL', 
+        success: function(r) {
+            appManager.isAdmin = r;
+            requestManager.ok(this);
```

Comment:
Sure, here's the user story: A user can can edit/delete only her comments. An admin can edit/delete any comment. We need to know beforehand if the current user is a normal user or an admin  (`user.authorizations.contains("ALL")`)